core/build_playerglobal/Cargo.toml

```diff
@@ -15,6 +15,6 @@ convert_case = "0.6.0"
 proc-macro2 = "1.0.79"
 quote = "1.0.35"
 swf = { path = "../../swf" }
-clap = {version = "4.5.4", features = ["derive"]}
+clap = {workspace = true, features = ["derive"]}
```

Comment:
Doesn't workspace declaration already set the ´derive` feature?

Reply:
I guess it does!

Reply:
This one is still there, no?

Reply:
Oh shoot the lack of a space made me miss it

Reply:
Maybe the same happened with `downcast-rs`, `fnv`, and `walkdir` then? :)

Reply:
Scratch that, my mistake.